kod/object/passive/spell/radiusench/bane.kod

```diff
@@ -0,0 +1,116 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+BaneEnchantment is RadiusEnchantment
+
+constants:
+
+   include blakston.khd
+
+resources:
+   
+   default_Hurt_Caster = "A tide of energy retaliates against your spell."
+   default_Kills_You = "The oncoming tide of energy carries you straight to "
+                       "the Underworld!"
+
+classvars:
+   
+   Bane_Hurt_Caster = default_Hurt_Caster
+   Bane_Kills_You = default_Kills_You
+
+   vrIcon = QorBane_name_rsc
+   vrDesc = QorBane_icon_rsc
+   vrName = QorBane_desc_rsc   
+
+   viSpell_num = SID_QORBANE
+   viSchool = SS_SHALILLE
+   viMana = 15
+   viSpell_level = 4
+   viSpellExertion = 5
+
+   viCast_time = 5000
+
+   viChance_To_Increase = 40
+   viMeditate_ratio = 30
+
+   viAffectsGuildmates = TRUE
+   viAffectsEnemies = TRUE
+   
+   viBanedSchool = SS_QOR
+   viRetaliationDamageType = ATCK_SPELL_HOLY
+
+properties:
+
+   piOldAreaEnchStyle = TRUE
+
+messages:
+
+   ResetReagents()
+   {
+      plReagents = $;
+      plReagents = Cons([&Yrxlsap,1],plReagents);
+      plReagents = Cons([&FairyWing,1],plReagents);
+
+      return;
+   }
+
+   GetDuration(iSpellPower=0)
+   {
+      local iDuration;
+
+      iDuration = 45 + (iSpellPower*3);  %% 30 - 228 seconds
+      iDuration = iDuration * 1000;      %% convert to seconds            
+
+      return random(iDuration/2,iDuration);
```

Comment:
I know we've only done this for buffs so far, but this would be a good time to remove the horrible random(iDuration/2,iDuration) effect sprinkled throughout the spell code for Shalbane/Qorbane.